src/server/server.js

```diff
@@ -197,6 +197,11 @@ Server.prototype._onSentryDown = function (sentryId) {
   var started = Date.now()
 
   nonblocking(presences).forEach(function (presence) {
+    // check if presence was destroyed
+    if (!presence || !presence.manager || !presence.manager.store) {
```

Comment:
Ok, sorry for the confusion, here's the whole deal:

The server destroys a resource (or basically removes all listeners etc) when there are no clients (subscribers local to that server) interested in it anymore.

A sentry-down however, causes the server to copy over the resources array using nonblocking, and asynchronously finds presences and if there are any remote clients belonging to the downed server, it sends offline messages to the locally subscribed-clients.

While this is happening, it's very much possible that the last local client interested in a presence can unsubscribe or go offline or simply get disconnected and go away. The server rightly will destroy such presences. 

The asynchronous sentry-down handling code, however, will continue in the background holding references to such resources. It may happen upon such a presence and read from the store which is in a dead state (since we no longer listen to Redis and keep it updated) and try to send messages to basically nobody. This is not consistent, and either way, it's wasted cycles. 

So let's skip these resources while processing sentry-down, however you think is appropriate. The above check is enough, but then you need to delete manager.store above. If not, you have to mark the resource somehow that it's been destroyed. (An unnecessary flag called "destroying" is already there for some reason).